apps/web/src/design-system/select/Select.styles.ts

```diff
@@ -28,6 +28,14 @@ export default createStyles((theme: MantineTheme, _params, getRef) => {
       top: '5px',
       margin: '2.5px 0px',
       lineHeight: '20px',
+      ['&[data-selected]']: {
```

Comment:
the `selected, hovered` props have been removed and now it should be implemented with `data-selected, data-hovered` attributes